code/src/index.css

```diff
@@ -1,13 +1,350 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+html {
+  height: 100%;
+}
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: #161616;
+  color: #fff;
+  min-height: 100%;
 }
-
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New", monospace;
+}
+header {
+  width: 100%;
+  display: block;
+  border-bottom: 1px solid #252525;
+  padding: 10px 20px 10px 20px;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 2;
+  background: #222222e3;
+  backdrop-filter: blur(12px);
+}
+h1 {
+  font-size: 18px;
+}
+.albums {
+  padding: 15px;
+  padding-top: 60px;
+  display:inline-block;
+  display: flex;
```

Comment:
You are overwriting value of `display` right after declaring it. This doesn't make a lot of sense, let's delete one of them